docs/api.rst

```diff
@@ -0,0 +1,10 @@
+===============
```

Comment:
I am not entirely sure about the purpose of this file, in my mind the simple thing would be to just include the simplesimdb module in index.rst somehow, or even better have the Reference to the two classes Manager and Repeater go directly on the main page. The "API Reference" just creates another level of indirection to click through...

Reply:
It took some experimentation, but I was able to achieve this in 9ceab46. It looks like the general layout of the docs had been copied across from another Python project that made use of many more modules, and in that case the indirection may have made more sense to avoid cluttering the main page.

Reply:
Perfect, yes this is what I wanted, I think from here I can figure out how to do more elaborate things in sphinx should it be necessary